workflow/scripts/mageck.py

```
@@ -3,40 +3,58 @@
+        "-k {count_table} "
+        "-t {test_sample} "
+        "-c {control_sample} "
+        "-n {dir_name}/{comparison} "
+        "{cnv} "
+        "{control} "
+        "{extra} "
+        "{log}"
+    )
+else:
+    # Run MAGeCK mle command
+    matrix = snakemake.input["matrix"]
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)